src/Compiler/Service/FSharpProjectSnapshot.fs

```diff
@@ -364,7 +364,10 @@ and internal ProjectCore
              |> Md5Hasher.addDateTimes (ReferencesOnDisk |> Seq.map (fun r -> r.LastModified))
              |> Md5Hasher.addBytes' (
                  ReferencedProjects
-                 |> Seq.map (fun (FSharpReference(_name, p)) -> p.ProjectSnapshot.SignatureVersion)
+                 |> Seq.map (function
+                     | FSharpReference(_name, p) -> p.ProjectSnapshot.SignatureVersion
+                     | PEReference(getStamp, _) -> Md5Hasher.empty |> Md5Hasher.addDateTime (getStamp ())
```

Comment:
Since it includes a `getStamp()`, does this mean that the hash will always be different and this never cached?